modules/cuda_plugin/build.sh

```diff
@@ -45,7 +45,8 @@ cmake "${OPENVINO_HOME}" \
       -DIE_EXTRA_MODULES="${OPENVINO_CONTRIB}/modules" \
       -DWHEEL_VERSION="${WHEEL_VERSION}" \
       -DVERBOSE_BUILD=ON \
-      -DCMAKE_BUILD_TYPE="${BUILD_TYPE}"
+      -DCMAKE_BUILD_TYPE="${BUILD_TYPE}" \
+      -DBUILD_cuda_plugin=OFF
```

Comment:
why do we need it?

Reply:
Its just for testing purposes, Im trying to figure out how to disable cuda plugin and see whether builds will pass